specifyweb/frontend/js_src/lib/components/Formatters/Components.tsx

```diff
@@ -266,7 +269,9 @@ export function ResourceMapping({
         })
       }
     >
-      <div>
+      <div
+        className={displayedFormatter === false ? 'flex flex-wrap gap-2' : ''}
```

Comment:
adding `flex flex-wrap gap-2` class name always seems fine - is this condition needed?

Reply:
also, this change along seems to fix the issue -why are any other changes in this PR necessary?

Reply:
but also, if you look at the code closer, the parent element of this div `Ul` already has the necessary class names (flex flex-wrap gap-2), and so this seems to suggest that the `<div>` was added here later and broke something

Reply:
and by looking at `git blame` for this file, you can see that the `<div>` was added in https://github.com/specify/specify7/commit/10ca4d8b5 as part of https://github.com/specify/specify7/pull/4114/files

Reply:
in https://github.com/specify/specify7/pull/4114#discussion_r1366110895 the screenshot shows that this was working correctly

Reply:
GitHub doesn't let me add a comment, but `${mappingLineProps.length > 1 ? 'flex' : ''}` seems not necessary - can simplify to always have `flex`

before:

![Screenshot 2024-01-12 at 16 30 33](https://github.com/specify/specify7/assets/40512816/a7ff7af3-cafb-4b02-b470-4558e11d2a9f)

after:

![Screenshot 2024-01-12 at 16 30 50](https://github.com/specify/specify7/assets/40512816/77d82c1e-6f18-489e-8099-ee3ee98451a3)